packages/hooks/src/useCreateWallet.ts

```diff
@@ -0,0 +1,41 @@
+import { useInvalidateQuery, useRequest } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+import { useMemo } from 'react';
+
+/** A custom hook to create more wallets */
```

Comment:
`non blocking suggestion:` perhaps we could make this more descriptive using JSDOc comments? something like this:

```
/**
     * A custom hook to create more wallets
     *
     * some description
     *
     * @returns {Object}
*/
```